### PR TITLE
Bug/Feature issue templates

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
### Description of Change ###

 

 ### Linked Issues ###
 

 - Fixes #

 ### PR Checklist ###
 
 - [ ] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [ ] Has samples (if omitted, state reason in description)
 - [ ] Rebased on top of `main` at time of PR
 - [ ] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui.Markup/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pulls/XYZ 

 ### Additional information ###

### CODE REVIEW

**Comment by michael-hawker** [comment]
I'd suggest asking questions at the end if this is something the submitter wants to help with or not. We've found this useful in the WCT repo (feel free to take a look at our templates there).

File: .github/ISSUE_TEMPLATE/bug_report_template.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    label: Link to public reproduction project repository
+    description: Please add a link to a public [reproduction project](https://github.com/dotnet/maui/blob/main/.github/repro.md) repository. Otherwise the issue will be marked with "Needs reproduction" label and will have a lower priority. Attached zip files cannot be opened by us.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      examples:
+        - **.NET MAUI CommunityToolkit**: 1.2.0
```
Comment (brminnick) [inline .github/ISSUE_TEMPLATE/bug_report_template.yml:45]:
```suggestion
        - **.NET MAUI C# Markup CommunityToolkit**: 1.2.0

```

File: .github/ISSUE_TEMPLATE/bug_report_template.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        - **OS**: Windows 10 Build 10.0.19041.0
+        - **.NET MAUI**: 6.0.408
+    value: |
+        - .NET MAUI CommunityToolkit:
```
Comment (brminnick) [inline .github/ISSUE_TEMPLATE/bug_report_template.yml:49]:
```suggestion
        - .NET MAUI C# Markup CommunityToolkit:
```

File: .github/ISSUE_TEMPLATE/open-a-new-feature-proposal.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    label: Detailed Design
+    description: This is the bulk of the proposal. Explain the design in enough detail for somebody familiar with .NET MAUI to understand, and for somebody familiar with the Community Toolkit to implement, and include examples of how the feature is used. Please include syntax and desired semantics for the change, including linking to the relevant parts of the existing .NET MAUI Toolkit spec to describe the changes necessary to implement this feature. An initial proposal does not need to cover all cases, but it should have enough detail to enable a community member to bring this proposal to design if they so choose.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Usage Syntax
+    render: markdown
+    description: Please provide an example of how an end-user will use this feature. If this is a UI control, please an example of the feature being consumed in both a C# UIand a XAML UI.
```
Comment (brminnick) [inline .github/ISSUE_TEMPLATE/open-a-new-feature-proposal.yml:59]:
```suggestion
    description: Please provide an example of how an end-user will use this feature. If this is a UI control, please an example of the feature being consumed in a C# UI.

```

**Review by brminnick — CHANGES_REQUESTED** [review]
Thanks Vlad! Just a few tweaks to focus on C# Markup and remove XAML

**Review by brminnick — APPROVED** [review]

**Pull request merged**